src/Components/WebAssembly/Build/src/targets/StaticWebAssetsWokarounds.targets

```
@@ -0,0 +1,47 @@
+<Project>
+  <!--
+    We're attempting to work around a behavior in the Razor SDK with publishing + static web assets for 3.1 applications.
+
+    Some assets such as dlls can appear both as part of a .NET project's publish output as well as the output of the wwwroot. For instance, consider the
+    MyApp.Shared.dll. The target in the RazorSDK: https://github.com/dotnet/aspnetcore-tooling/blob/fae9a5fdbc19edcf5b59a093ce347897a78b0386/src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.StaticWebAssets.targets#L493-L519
+    unconditionally retargets these items to the wwwroot sub-directory of the publish folder.
+
+    The behavior we want is for these items to appear in both places.
+  -->
+  <Target
+    Name="_BlazorStashStaticWebAssets"
+    BeforeTargets="_StaticWebAssetsComputeFilesToPublish"
+    Condition="'$(RazorSdkSupportPublishingSharedBinaries)' == ''">
```

Should this be something like:

```suggestion
    Condition="'$(RazorSdkSupportPublishingSharedBinaries)' != 'false'">
```

Otherwise, it seems odd that the values `true` and `false` should produce the same behavior.

Same on L40

---

Also should this have an underscore-prefixed pubternal name? We don't want this API to stick around forever, as it ceases to be meaningful when we fix the underlying SDK issue.